.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This won't take effect for these's no corresponding github actionshttps://github.com/marketplace/actions/pr-title-checker

Reply:
deleted